indico/modules/events/papers/client/js/components/RevisionJudgment.jsx

```diff
@@ -31,85 +31,87 @@ export default function RevisionJudgment({revision}) {
   }
 
   return (
-    <div className="i-timeline-item">
-      <UserAvatar user={judge} />
-      <div className="i-timeline-item-box header-indicator-left">
-        <div className="i-box-header flexrow">
-          <div className="f-self-stretch">
-            {state === PaperState.accepted && (
-              <Translate>
-                <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> accepted this
-                paper.
-              </Translate>
-            )}
-            {state === PaperState.rejected && (
-              <Translate>
-                <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> rejected this
-                paper.
-              </Translate>
+    <>
+      <div className="i-timeline-item">
+        <UserAvatar user={judge} />
+        <div className="i-timeline-item-box header-indicator-left">
+          <div className="i-box-header flexrow">
+            <div className="f-self-stretch">
+              {state === PaperState.accepted && (
+                <Translate>
+                  <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> accepted
+                  this paper.
+                </Translate>
+              )}
+              {state === PaperState.rejected && (
+                <Translate>
+                  <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> rejected
+                  this paper.
+                </Translate>
+              )}
+              {state === PaperState.to_be_corrected && (
+                <Translate>
+                  <Param name="judgeName" value={judge.fullName} wrapper={<strong />} /> asked for
+                  changes.
+                </Translate>
+              )}{' '}
+              <time dateTime={serializeDate(judgmentDt, moment.HTML5_FMT.DATETIME_LOCAL)}>
+                {serializeDate(judgmentDt, 'LL')}
+              </time>
+            </div>
+            {canJudge && isLastRevision && (
+              <>
+                <div className="hide-if-locked">
```

Comment:
if `canJudge` takes the event being locked into account we don't need this (it's just a quick way to hide action buttons etc. in locked events without the need for explicit checks or JS)